torch/lib/c10d/TCPStore.hpp

```diff
@@ -74,6 +75,8 @@ class TCPStore : public Store {
 
   void set(const std::string& key, const std::vector<uint8_t>& value) override;
 
+  std::vector<uint8_t> compareSet(const std::string& key, const std::vector<uint8_t>& currentValue, const std::vector<uint8_t>& newValue);
```

Comment:
nitpick: Assuming we have some code formatting guidelines, what are the hard/soft limits on line lengths? This line (also the one in PrefixStore.hpp) seemed too long to me. And a follow-up question: do we have any sort of tooling (e.g. `clang-format`) that enforces those guidelines?

Reply:
we do have clang-format, clang-tidy, and githooks, but I'm not 100% how/when they are run. Will double check this.